database/migrations/2025_05_19_000000_create_game_titles_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::create('game_titles', function (Blueprint $table) {
```

Comment:
Looking forward, I wonder if this should be 'game_releases' where we could have regional release dates in addition to regional titles.

Reply:
I think this is an excellent idea.

This prompted a whole bunch of changes into the PR, but I've implemented it in latest.

"Titles" has been changed to "Releases":
![Screenshot 2025-05-21 at 8 31 18 PM](https://github.com/user-attachments/assets/f6a624c4-4dc5-4068-bc11-79915392c693)

Additionally, `GameData.released_at` and `GameData.released_at_granularity` are now treated as denormalized values, derived from the new `game_releases` table. The values in `GameData` automatically update according to whatever is in the `->releases()` relation.

Accordingly, the "First Released At" section in `GameResource` has been removed.